models_serde.go

```diff
@@ -1,20 +1,47 @@
 //go:build go1.18
 // +build go1.18
 
-// Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.9.0, generator: @autorest/go@4.0.0-preview.42)
+// Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.9.3, generator: @autorest/go@4.0.0-preview.43)
```

Comment:
Pl make sure all the marshal/UnMarshal exist as before. I see lot of deletions so cant check.

Also for similiciy, Use preview.42 and GoLang18. So that diffs are minimized.

Reply:
As discussed offline, will not use GoLang 1.8 as its a older version for which support has ended.
For deletions - have reverified that none of the existing marshall/unmarshall methods have actually gotten deleted - they have just been moved up or down in the same file so closing this comment